package.json

```diff
@@ -47,6 +47,9 @@
     "express": "^4.17.1",
     "http-status-codes": "^2.1.4",
     "joi": "^17.4.0",
+    "pg": "^8.5.1",
```

Comment:
Please, use fixed package versions

Reply:
Changed to the fixed versions. Thank you/